src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import services from '../services';
+
+const { findUser } = services;
+
+const secret = process.env.SECRET_KEY;
+
+/**
+ *
+ *
+ * @param {object} req
+ * @param {object} res
+ * @param {*} next
+ * @returns {*} json or next
+ */
+const verifyToken = (req, res, next) => {
+  const token = req.headers.authorization;
+  if (!token) {
+    return res.status(401).json({
+      error: 'No token provided'
+    });
+  }
+  jwt.verify(token, secret, async (err, decoded) => {
+    if (err) {
+      return res.status(401).json({
+        error: 'Failed to authenticate token'
```

Comment:
Hey, we should use the default JWT error message here (`err.message`) instead of this custom one.
I think the default message gives more context to **why** the server "_failed to authenticate token_", for example, _invalid token_, _token expired,_ etc.
What do you think?

Reply:
Yeah makes sense

Reply:
But the error messages are not descriptive enough. its either 'invalid signature' or 'jwt malformed' which both fall under authentication failure